server/php/UploadHandler.php

```diff
@@ -367,7 +367,7 @@ protected function validate($uploaded_file, $file, $error, $index) {
             return false;
         }
         $content_length = $this->fix_integer_overflow(
-            (int)$this->get_server_var('CONTENT_LENGTH')
+            $this->get_server_var('CONTENT_LENGTH')
```

Comment:
Can you confirm that with this change, the later comparison checks are still testing numbers against each other?
e.g. the check [here](https://github.com/servocoder/jQuery-File-Upload/blob/master/server/php/UploadHandler.php#L373) and all later file size comparisons after setting file size [here](https://github.com/servocoder/jQuery-File-Upload/blob/master/server/php/UploadHandler.php#L384) for non-multipart uploads.